Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -298,51 +211,144 @@ internal List<Result> Search(IEnumerable<Result> results, string searchName)
                 {
                     var matchResult = StringMatcher.FuzzySearch(searchName, x.Title);
                     if (matchResult.IsSearchPrecisionScoreMet())
-                        x.Score = matchResult.Score;
+                        x.Score += matchResult.Score;
```

Comment:
'+' ? did it get assigned score somewhere else as well?

Reply:
Hmmm I remember windows setting contain something like this. Not sure why I add a + here. I will check later.

Reply:
Oh it is used to bump updatable plugin score for `pm list`